$MirOS: ports/devel/autoconf/2.63/patches/patch-lib_m4sugar_m4sh_m4,v 1.1 2009/07/15 16:43:36 bsiegert Exp $
--- lib/m4sugar/m4sh.m4.orig	Wed Aug 20 14:46:30 2008
+++ lib/m4sugar/m4sh.m4	Tue Nov 17 14:10:35 2009
@@ -1215,13 +1215,14 @@ _ASBOX])
 # m4_text_wrap hacks^Wworks around the fact that m4_format does not
 # know quadrigraphs.
 #
+# Old definition from autoconf 2.61 -BS
+m4_define([AS_PadLeft],
+	   [m4_if(m4_eval([$1] > [$2]),1,[[ ]AS_PadLeft([$1]-1,[$2])])])
 m4_define([AS_HELP_STRING],
-[m4_text_wrap([$2], m4_cond([[$3]], [], [                          ],
-			    [m4_eval([$3]+0)], [0], [[$3]],
-			    [m4_format([[%*s]], [$3], [])]),
-	      m4_expand([  $1 ]), [$4])dnl
+[m4_pushdef([AS_Prefix], m4_default([$3], [                          ]))dnl
+m4_text_wrap([$2], AS_Prefix, [  $1]AS_PadLeft(m4_len(AS_Prefix),m4_len([$1])+2))dnl
+m4_popdef([AS_Prefix])dnl
 ])# AS_HELP_STRING
-
 
 # AS_IDENTIFIER_IF(EXPRESSION, IF-IDENT, IF-NOT-IDENT)
 # ----------------------------------------------------
